core/services/relay/evm/types/gen/main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"bytes"
+	_ "embed"
+	"go/format"
+	"os"
+	"text/template"
+)
+
+func main() {
+	genInts()
+	genBytes()
+}
+
+func genBytes() {
+	byteTypes := [33]ByteType{}
+	for i := 1; i < 33; i++ {
+		byteTypes[i-1].Size = i
+	}
+	runTemplate("bytes", bytesTemplate, "byte_types_gen.go", byteTypes)
+}
+
+func genInts() {
+	var intTypes []*IntType
+
+	// 8, 16, 32, and 64 bits have their own type in go that is used by abi.
+	// The test use *big.Int
+	for i := 24; i <= 256; i += 8 {
+		if i == 32 || i == 64 {
+			continue
+		}
+
+		signed := &IntType{Size: i, Signed: false}
+		unsigned := &IntType{Prefix: "u", Size: i}
+		intTypes = append(intTypes, signed, unsigned)
+	}
+	runTemplate("ints", intsTemplate, "int_types_gen.go", intTypes)
+}
+
+func runTemplate(name, rawTemplate, outputFile string, input any) {
+	t, err := template.New(name).Parse(rawTemplate)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
`package template` has `Must` for this case:
```suggestion
	t := template.Must(template.New(name).Parse(rawTemplate))
```
Maybe it is better to explicitly panic in order to align with the cases below, but how about renaming so it is obvious it may panic? e.g. `func mustRunTemplate()`